examples/spark/images/base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:latest
+FROM java:openjdk-8-jdk
```

Comment:
(And I didn't even push a new image, because `:latest` is actually == `:openjdk-8-jdk` right now.)